File: Lab2/app/src/main/java/com/example/firstjavaapp/models/Image.java
package com.example.firstjavaapp.models;

import java.util.ArrayList;

public class Image {

    public ArrayList<DocImage> docs;
    public int total;
    public int limit;
    public int page;
    public int pages;

    @Override
    public String toString(){
        return this.docs.get(0).previewUrl + " , высота: " + this.docs.get(0).height;
    }
}

File: Lab_2_NEW/app/src/main/java/com/example/lab_2_new/models/Image.kt
package com.example.lab_2_new.models

class Image {
    var docs: ArrayList<DocImage>? = null
    var total = 0
    var limit = 0
    var page = 0
    var pages = 0

    override fun toString(): String {
        return docs!![0].previewUrl.toString() + " , высота: " + docs!![0].height
    }
}